text/0000-Project-Goals-2024h2.md

```diff
@@ -0,0 +1,522 @@
+| ---                                                                                                | ---             | --- |
+| *Discussion and moral support*                                                                     |                 |  |
+| ↳ [Make Rustdoc Search easier to learn](https://rust-lang.github.io/rust-project-goals/2024h2/rustdoc-search.html#ownership-and-team-asks)                 | [Michael Howell][]      |  |
+| *RFC decision*                                                                                     |                 |  |
+| ↳ [Stabilize doc_cfg](https://rust-lang.github.io/rust-project-goals/2024h2/doc_cfg.html#ownership-and-team-asks)                                          | [Guillaume Gomez][] |  |
+| *Standard reviews*                                                                                 |                 |  |
+| ↳ [Stabilize doc_cfg](https://rust-lang.github.io/rust-project-goals/2024h2/doc_cfg.html#ownership-and-team-asks)                                          | [Guillaume Gomez][] |  |
+| ↳ [Implement "merged doctests" to save doctest time](https://rust-lang.github.io/rust-project-goals/2024h2/merged-doctests.html#ownership-and-team-asks)   | [Guillaume Gomez][] |  |
+
+### rustdoc-frontend team
+| Goal                                                                                  | Owner      | Notes |
+| ---                                                                                   | ---        | --- |
+| *Design meeting*                                                                      |            |  |
+| ↳ [Improve on any discovered weaknesses](https://rust-lang.github.io/rust-project-goals/2024h2/rustdoc-search.html#ownership-and-team-asks)   | [Michael Howell][] |  |
+| *FCP review*                                                                          |            |  |
+| ↳ [Improve on any discovered weaknesses](https://rust-lang.github.io/rust-project-goals/2024h2/rustdoc-search.html#ownership-and-team-asks)   | [Michael Howell][] |  |
+| *Standard reviews*                                                                    |            |  |
+| ↳ [Improve on any discovered weaknesses](https://rust-lang.github.io/rust-project-goals/2024h2/rustdoc-search.html#ownership-and-team-asks)   | [Michael Howell][] |  |
+
+### types team
+| Goal                                                                                                | Owner            | Notes              |
+| ---                                                                                                 | ---              | ---                |
+| *Design meeting*                                                                                    |                  |                    |
+| ↳ [Experiment with relaxing the Orphan Rule](https://rust-lang.github.io/rust-project-goals/2024h2/Relaxing-the-Orphan-Rule.html#ownership-and-team-asks)   | ![Help wanted][] | Up to 1, if needed |
```

Comment:
This is the one project goal I feel somewhat uncomfortable with. I feel like I (and the wider types team) are already at capacity with the existing project goals and any additional bugs, unsoundnesses, or nominations during the next 6 months.

I feel especially uncertain as the required team input is very dependent on the owner and the direction they take this in. I don't trust that a single design meeting is sufficient, given that it will also require review work and asynchronous discussion. 

Even if we do not accept it, people can still work on it. The teams should be able to provide some feedback even for non-project goals. Having it be a project goal does worry me as I expect it to increase the pressure on team members to quickly respond to questions and proposals here.

Reply:
This is a solid point. I agree that the number of goals grew towards the end (initially there were few) and that we've got plenty to do. When thinking about this goal I didn't think properly about the *types* team interaction. This is likely a poor fit for an orphaned goal in particular as that implies we are looking to find owners for the external community, which probably works best when non-specialized knowledge is required.